src/configure/configure.ts

```diff
@@ -451,9 +488,32 @@ class PipelineConfigurer {
                 .then((webApps) => webApps.map(x => { return { label: x.name, data: x }; })),
             { placeHolder: Messages.selectWebApp },
             TelemetryKeys.WebAppListCount);
+
+        await this.validateIfPipelineCanBeSetupOnResource(selectedResource.data.id);
         this.inputs.targetResource.resource = selectedResource.data;
     }
 
+    private async updateScmType(queuedPipeline: Build) {
+        await this.appServiceClient.updateScmType(this.inputs.targetResource.resource.id);
+
+        let metadata = await this.appServiceClient.getSiteMetadata(this.inputs.targetResource.resource.id);
+        let organizationId = await this.azureDevOpsClient.getOrganizationIdFromName(this.inputs.organizationName);
+        metadata["properties"] = {
+            "VSTSRM_ProjectId": `${this.inputs.project.id}`,
+            "VSTSRM_AccountId": `${organizationId}`,
+            "VSTSRM_BuildDefinitionId": `${queuedPipeline.definition.id}`,
+            "VSTSRM_BuildDefinitionWebAccessUrl": `${queuedPipeline.definition._links.web.href}`,
```

Comment:
deployment center is only able to understand old .visualstudio.com url formats. Thus we need to do these steps
1. Convert dev.azure.com urls to old visualstudio.com urls
2. Later update deployment center to handle both types of url
3. After that remove the url conversion from here.

Reply:
I have done the first step and the deployment center view lights up successfully.